identityClient/api.go

```diff
@@ -1120,6 +1120,19 @@ type GetFederatedIdentitiesForSyncRequest struct {
 	PrimaryRealmEndpoint string
 }
 
+// TotpMFASettings wraps the information about an Identity's MFA settings, if any.
+type TotpMFASettings struct {
+	ID             string `sql:"id"`
```

Comment:
`json:"id"` here and elsewhere

Reply:
Oops! I was working in identity's db so I didn't context switch 😓